### DISCUSSION TITLE
kpanda/user-guide/clusters/create-cluster/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1356] [Announcements]
# kpanda/user-guide/clusters/create-cluster/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/clusters/create-cluster/

**Comment by liyuerich** [comment]
前提条件:k8s版本和图片中的不一致

**Reply by lemontrees123** [reply]
已更正

**Comment by liyuerich** [comment]
前提条件中能否提供具体命令帮助客户确认是否符合条件？
